### ISSUE TITLE
Modules missing from react-router-dom beta.2

### ISSUE THREAD

**Issue by kgoggin** [ReactTraining/react-router#4397]
## Version
4.0.0-beta.2

## Steps to reproduce
```javascript
npm install react-router-dom@next
```

## Expected Behavior
Able to `import` modules per docs.
## Actual Behavior
Not all modules listed in index.js are included with the package. It appears that any of the modules that actually reside in the react-router root package aren't being bundled w/ react-router-dom. But that's an issue because react-router-dom's index.js references them like so:

```javascript
var _MemoryRouter2 = require('./MemoryRouter');
```

**Comment by mjackson** [comment]
We fixed this already :) https://github.com/ReactTraining/react-router/pull/4391

**Comment by kgoggin** [comment]
Ah, my bad, forgot to search first. ETA on the release?

**Comment by YasserKaddour** [comment]
If you are eager to use the beta; you can move the missing file from `node_modules/react-router` to `node_modules/react-router-dom`, and renaming `ServerRouter.js` to `StaticRouter.js`

**Comment by mjackson** [comment]
Published in 4.0.0-beta.3. `npm install react-router-dom@next`

**Comment by mdabutaleb** [comment]
awesome :) it's working for me.
